### DISCUSSION TITLE
Having issue with my onedrive index

### DISCUSSION THREAD

**Discussion by IraaUchiha** [spencerwooo/onedrive-vercel-index#625] [Q&A]
it was working fine some days ago now i checked and i saw this

![image](https://user-images.githubusercontent.com/91321234/165923771-8ea4edd8-5959-4206-b69e-44c2536f1c30.png)

P.S. i tried to redeploying the index in vercel but the thing stayed the same

**Comment by myl7** [comment]
Is it convenient for you to provide the URL / repo? The log can barely help to locating the bug.
